doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -593,6 +607,7 @@ weight can be set separately from the figure title using :rc:`figure.labelsize`
 and :rc:`figure.labelweight`.
 
 .. plot::
+    :alt: A figure with 4 plots organised in 2 rows and 2 columns. The title of the figure is suptitle in bold and 64 points. The x axis is labelled supxlabel, and y axis is labelled subylabel. Both labels are 32 points and bold
```

Comment:
```suggestion
    :alt: A figure with 4 plots organised in 2 rows and 2 columns. The title of the figure is suptitle in bold and 64 points. The x axis is labelled supxlabel, and y axis is labelled subylabel. Both labels are 32 points and bold.
```

Reply:
Should the numbers be spelled out as words?